packages/dds/map/src/interfaces.ts

```diff
@@ -21,6 +22,70 @@ export interface IValueChanged {
     previousValue: any;
 }
 
+/**
+ * Operation indicating a value should be set for a key.
+ */
+ export interface IMapSetOperation {
```

Comment:
I like moving these to a dedicated interfaces file, but probably shouldn't be exported from the package (by being selective in index.ts about what to export).

One thing you might try to see how it looks would be to create separate publicInterfaces.ts vs. internalInterfaces.ts.  I don't think we do this elsewhere, don't know if it would work out well vs. just be annoying, but something to consider.

Reply:
I'm going to try that